src/gpu/generic/sycl/matmul_kernels.hpp

```diff
@@ -409,16 +410,23 @@ struct matmul_kernel_fwd_t {
                   CTX_IN_SYCL_KERNEL_MEMORY(DNNL_ARG_ATTR_DROPOUT_PROBABILITY))
         , po_args_(cgh, ctx, conf_.post_ops) {}
 
-    void operator()(::sycl::nd_item<1> item) const {
+    void operator()(::sycl::nd_item<1> item, ::sycl::kernel_handler kh) const {
```

Comment:
```suggestion
    void operator()(::sycl::nd_item<1> item, const ::sycl::kernel_handler &kh) const {
```
?

Reply:
The kernel_handler cannot be const because the `get_specialization_constant()` is not marked caused and would throw an error.